syft/frameworks/crypten/model.py

```diff
@@ -0,0 +1,194 @@
+import functools
```

Comment:
This is **not used** at the moment.
I want to create an object that plays the role of a store -- for a model that is converted with ```onnx```